include/alp/algorithms/symm_tridiag_eigensolver.hpp

```diff
@@ -117,29 +124,25 @@ namespace alp {
 		 *
 		 */
 		template<
-			typename D,
-			typename VecView1,
-			typename VecImfR1,
-			typename VecImfC1,
-			typename VecView2,
-			typename VecImfR2,
-			typename VecImfC2,
-			typename VecView3,
-			typename VecImfR3,
-			typename VecImfC3,
-			typename OrthogonalType,
-			typename OrthViewType,
-			typename OrthViewImfR,
-			typename OrthViewImfC,
-			class Ring = Semiring< operators::add< D >, operators::mul< D >, identities::zero, identities::one >,
+			typename VectorEgVals,
+			typename VectorD,
+			typename VectorV,
+			typename OrthogonalMat,
```

Comment:
Should there be a static assert about Mat  being an ALP matrix + Orthogonal (eg, based on `is_matrix< OrthogonalMat >` + `is_a< OrthogonalMat, Orthogonal >` ?

Reply:
Sorry, I did not see these comments. They are addressed in PR  #112